lti/lti-tool/src/webapp/vm/lti_tool_edit.vm

```diff
@@ -254,6 +254,22 @@ $(document).ready( function() {
 });
 
 #if ($isEdit && $isAdmin && $autoStartUrl )
+function importLTI13Custom(customObject) {
+    if (typeof customObject != "object") return "";
+    customString = "";
+    for (const key in customObject) {
+        if ( typeof key != "string" ) continue;
+        value = customObject[key];
+        if ( typeof value != "string" ) continue;
+        console.log(key + ": " + value);
```

Comment:
Remove

Reply:
I assume you mean the console.log - I agree.